core/templates/pages/notifications-dashboard-page/notifications-dashboard-page.component.ts

```diff
@@ -16,44 +16,61 @@
  * @fileoverview Component for the user's notifications dashboard.
  */
 
+import { Component } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { WindowRef } from 'services/contextual/window-ref.service';
+import { DateTimeFormatService } from 'services/date-time-format.service';
+import { LoaderService } from 'services/loader.service';
+import { NotificationsDashboardPageBackendApiService } from './notifications-dashboard-page-backend-api.service';
 require('base-components/base-content.directive.ts');
```

Comment:
why this require is still here?

Reply:
To register base-content directive with angular js, otherwise, top navigation bar and footer (which are still in ajs) are not getting properly rendered as base-content is considered a normal html5 tag.